lib/Account.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Stripe;
+
+class Account extends ApiResource
+{
+    public function instanceUrl()
+    {
+        if ($this['id'] === null) {
+            return '/v1/account';
+        } else {
```

Comment:
Subjective style decision, either way is practically fine (unless PSR-2 has an opinion here).